tools/firmware-utils/src/mkfwimage.c

```diff
@@ -128,6 +128,15 @@ struct fw_info fw_info[] = {
 		},
 		.sign = true,
 	},
+	{
+		.name = "ACB-ISP",
+		.fw_layout = {
+			.kern_start	=	0x9f050000,
+			.kern_entry	=	0x80002000,
+			.firmware_max_length=	0x00F60000,
+		},
+		.sign = true,
+	},
```

Comment:
This is exactly the same as "WA". Is this correct?
I assume we would still want a separate definition, as WA is another SOC?

Reply:
It is correct that it is the same as WA. Problem is that the UBNT_BOARD is used for that. It would be very odd and maybe misleading at some places if the UBNT_BOARD variable is set to WA for an aircube.

Reply:
Yes, so keep it as you did it.